*
* Visual Studio Code Server
*
* By using the software, you agree to
* the Visual Studio Code Server License Terms (https://aka.ms/vscode-server-license) and
* the Microsoft Privacy Statement (https://privacy.microsoft.com/en-US/privacystatement).
*
Server bound to 127.0.0.1:36807 (IPv4)
Extension host agent listening on 36807

[12:56:15] 




[12:56:15] Extension host agent started.
[12:56:15] [127.0.0.1][10b49f82][ManagementConnection] New connection established.
[12:56:15] [127.0.0.1][16817b1b][ExtensionHostConnection] New connection established.
[12:56:15] [127.0.0.1][16817b1b][ExtensionHostConnection] <1875084> Launched Extension Host Process.
New EH opened, aborting shutdown
[13:01:15] New EH opened, aborting shutdown
[17:33:51] [127.0.0.1][90eed2fd][ManagementConnection] New connection established.
[17:33:51] [127.0.0.1][0f97378a][ExtensionHostConnection] New connection established.
[17:33:51] [127.0.0.1][0f97378a][ExtensionHostConnection] <2357323> Launched Extension Host Process.
rejected promise not handled within 1 second: CodeExpectedError: Could not find pty on pty host
stack trace: CodeExpectedError: Could not find pty on pty host
    at L.U (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:43:32314)
    at L.updateTitle (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:43:27475)
    at W.<computed> (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:43:22722)
    at Object.call (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:34:5005)
    at c.s (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:32:5118)
    at c.q (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:32:4641)
    at g.value (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:32:4052)
    at r.y (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:27:1902)
    at r.fire (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:27:2119)
    at process.le (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:25:9948)
    at process.emit (node:events:513:28)
    at emit (node:internal/child_process:937:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[17:33:54] Error [CodeExpectedError]: Could not find pty on pty host
    at L.U (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:43:32314)
    at L.updateTitle (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:43:27475)
    at W.<computed> (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:43:22722)
    at Object.call (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:34:5005)
    at c.s (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:32:5118)
    at c.q (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:32:4641)
    at g.value (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:32:4052)
    at r.y (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:27:1902)
    at r.fire (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:27:2119)
    at process.le (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:25:9948)
    at process.emit (node:events:513:28)
    at emit (node:internal/child_process:937:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[17:48:06] [127.0.0.1][10b49f82][ManagementConnection] The client has disconnected gracefully, so the connection will be disposed.
[17:48:06] [127.0.0.1][16817b1b][ExtensionHostConnection] <1875084> Extension Host Process exited with code: 0, signal: null.
[17:48:22] [127.0.0.1][abf86d33][ManagementConnection] New connection established.
[17:48:22] [127.0.0.1][31197bd6][ExtensionHostConnection] New connection established.
[17:48:22] [127.0.0.1][31197bd6][ExtensionHostConnection] <2362145> Launched Extension Host Process.
rejected promise not handled within 1 second: CodeExpectedError: Could not find pty on pty host
stack trace: CodeExpectedError: Could not find pty on pty host
    at L.U (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:43:32314)
    at L.updateTitle (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:43:27475)
    at W.<computed> (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:43:22722)
    at Object.call (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:34:5005)
    at c.s (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:32:5118)
    at c.q (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:32:4641)
    at g.value (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:32:4052)
    at r.y (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:27:1902)
    at r.fire (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:27:2119)
    at process.le (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:25:9948)
    at process.emit (node:events:513:28)
    at emit (node:internal/child_process:937:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[17:48:24] Error [CodeExpectedError]: Could not find pty on pty host
    at L.U (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:43:32314)
    at L.updateTitle (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:43:27475)
    at W.<computed> (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:43:22722)
    at Object.call (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:34:5005)
    at c.s (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:32:5118)
    at c.q (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:32:4641)
    at g.value (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:32:4052)
    at r.y (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:27:1902)
    at r.fire (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:27:2119)
    at process.le (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:25:9948)
    at process.emit (node:events:513:28)
    at emit (node:internal/child_process:937:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[19:30:29] [127.0.0.1][abf86d33][ManagementConnection] The client has disconnected gracefully, so the connection will be disposed.
[19:30:29] [127.0.0.1][31197bd6][ExtensionHostConnection] <2362145> Extension Host Process exited with code: 0, signal: null.
[19:32:07] [127.0.0.1][bb2e7309][ManagementConnection] New connection established.
[19:32:07] [127.0.0.1][2fc45519][ExtensionHostConnection] New connection established.
[19:32:07] [127.0.0.1][2fc45519][ExtensionHostConnection] <2547430> Launched Extension Host Process.
rejected promise not handled within 1 second: CodeExpectedError: Could not find pty on pty host
stack trace: CodeExpectedError: Could not find pty on pty host
    at L.U (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:43:32314)
    at L.updateTitle (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:43:27475)
    at W.<computed> (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:43:22722)
    at Object.call (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:34:5005)
    at c.s (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:32:5118)
    at c.q (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:32:4641)
    at g.value (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:32:4052)
    at r.y (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:27:1902)
    at r.fire (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:27:2119)
    at process.le (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:25:9948)
    at process.emit (node:events:513:28)
    at emit (node:internal/child_process:937:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[19:32:09] Error [CodeExpectedError]: Could not find pty on pty host
    at L.U (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:43:32314)
    at L.updateTitle (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:43:27475)
    at W.<computed> (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:43:22722)
    at Object.call (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:34:5005)
    at c.s (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:32:5118)
    at c.q (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:32:4641)
    at g.value (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:32:4052)
    at r.y (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:27:1902)
    at r.fire (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:27:2119)
    at process.le (/home/agu078/.vscode-server/bin/2b35e1e6d88f1ce073683991d1eff5284a32690f/out/vs/platform/terminal/node/ptyHostMain.js:25:9948)
    at process.emit (node:events:513:28)
    at emit (node:internal/child_process:937:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
